packages/grafana-runtime/src/services/pluginExtensions/links.test.ts

```diff
@@ -0,0 +1,61 @@
+import { getPluginLink, PluginLinkMissingError } from './links';
+import { setPluginsExtensionRegistry } from './registry';
+
+describe('getPluginLink', () => {
+  describe('when getting a registered extension link', () => {
+    const pluginId = 'grafana-basic-app';
+    const linkId = 'declare-incident';
+
+    beforeAll(() => {
+      setPluginsExtensionRegistry({
+        links: {
+          [`${pluginId}.${linkId}`]: {
+            description: 'Declaring an incident in the app',
+            href: `/a/${pluginId}/declare-incident`,
```

Comment:
Question: wasn't the prop named `path` [in the `PluginsExtensionLinkConfig` type above](https://github.com/grafana/grafana/pull/61663/files#diff-969e280743cb22049967d8f6cd9b61a7337b5c4b8dd888389bbfa67d8134428dR10)? 🤔

Reply:
I think we have been consistent with: 

- `path` = the relative path to the plugins root.
- `href` = the absolute url to the exposed plugin page.